lesson4/hometask_tohichU.rb

```diff
@@ -0,0 +1,23 @@
+class LazyHash < Hash
+
+  def method_missing(method)
```

Comment:
Use all possible parameters here: `method_missing (method, *args, &block)`
All of them should be addressed in `super` in case when we don't have key in hash.